README.md

```diff
@@ -24,17 +24,21 @@ Before proceeding, make sure that there are no current builds in the `Build` ste
    
 3. Repeat steps 1 and 2 for the [docs-generated repo](https://github.com/OpenLiberty/docs-generated).
 
-4. Create a new branch off of the `prod` branch of this repo to add the new release version to the `branches` section of the [`prod playbook`](https://github.com/OpenLiberty/docs-playbook/blob/prod/antora-playbook.yml). Make a pull request into the `prod branch`. Once the pull request has been reviewed merge it in.
+4. Create a new branch off of the `prod` branch of this repo to add the new release version to the `branches` section of the [`prod playbook`](https://github.com/OpenLiberty/docs-playbook/blob/prod/antora-playbook.yml). Ensure that the branch name added matches the branch name in Github (e.g. includes a 'v' at the start). Make a pull request into the `prod branch`. Once the pull request has been reviewed merge it in.
```

Comment:
should we say "Create a pull request from your original branch into `prod branch` instead of Make a pull request into the `prod branch` ?

Reply:
Actually @aknguyen7 these wordings are as per prod branch. If we want to make change we can ask @kinueng or @mbroz2. My change here in Readme file is 8th point only

Reply:
i was a little confuse when i read it the first time